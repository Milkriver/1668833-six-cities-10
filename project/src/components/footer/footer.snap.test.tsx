import {render} from '@testing-library/react';
import Footer from './footer';

describe('Component: Footer', () => {
  it('should render correctly', () => {
    const {container} = render(<Footer />);

    expect(container).toMatchSnapshot();
  });
});
