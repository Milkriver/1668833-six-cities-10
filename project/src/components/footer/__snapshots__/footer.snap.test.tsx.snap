// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Footer should render correctly 1`] = `
<div>
  <footer
    class="footer container"
  >
    <a
      class="footer__logo-link"
      href="main.html"
    >
      <img
        alt="6 cities logo"
        class="footer__logo"
        height="33"
        src="img/logo.svg"
        width="64"
      />
    </a>
  </footer>
</div>
`;
